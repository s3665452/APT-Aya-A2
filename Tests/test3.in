RBYLRYRURYULUYRUBLBLBRULBLLUYUYBUBYBUYULURURUULURBRBBYLLLLYYURBYRBRYRLRYRURYRYLLBLBYRLBYBBBULRLYBUUY
Jon
Robb
turn 1 B 1
turn 4 U 2
turn 0 Y 3
turn 2 R 3
turn 5 B 2
turn 0 L 5
turn 3 U 4
turn 0 U 4
turn 0 R 5
