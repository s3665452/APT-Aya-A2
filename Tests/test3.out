Factories:
0: Y Y L
1: 
2: 
3: 
4: 
5: 

Score for Jon: 0
Mosaic for Jon:
1:         B || . . . . .
2:       B B || . . . . .
3:     . Y Y || . . . . .
4:   . . . U || . . . . .
5: . . R R R || . . . . .
broken:

Score for Robb: 0
Mosaic for Robb:
1:         . || . . . . .
2:       U U || . . . . .
3:     . R R || . . . . .
4:   . U U U || . . . . .
5: . . L L L || . . . . .
broken: F
